src/main.js

```diff
@@ -14,7 +14,10 @@ window.onload = () => {
   }
 
   let checkIsPageWithPlayer = document.querySelector("div.inline_player");
-  if (checkIsPageWithPlayer && window.location.href != "https://bandcamp.com/") {
+  if (
+    checkIsPageWithPlayer &&
+    window.location.href != "https://bandcamp.com/"
```

Comment:
This extension is already scoped for this URL, so is checking for this href necessary? What about `http://`? Looks like they 301 to `https://` urls, so does the extension actually need permissions for `http://*.bandcamp.com/*`?

Reply:
@emcniece this prevents the plugin from messing with the wrong pages.

Notice, it's specifically `!=`. This is because music pages always start with <bandname>.bandcamp.com. So this check makes sure it doesn't load on any other page (lots of bandcamp pages have a player but should not be modified)